performance_metrics/tutorial.md

```diff
@@ -0,0 +1,142 @@
+# Performance metrics
+
+Gazebo 9 and Gazebo 11 publish a message called `/gazebo/performance_metrics` that allows to check the performance
+of each sensor in the world.
+
+# Learn how to read the measurements
+
+We are going to compare the results between physics (contact) and rendering (camera) based sensors and the differences
+when the `lockstep` flag is active.
+
+## Camera
+
+For the rendering-based sensor we are going to use the [camera_strict_rate.world](https://github.com/osrf/gazebo/blob/gazebo9/test/worlds/camera_strict_rate.world) world. It contains two cameras: a high-resolution (1280x720) high-frame-rate (500 fps) and a a low-resolution (320x240) low-frame-rate (30 fps) camera.
+
+### Run the world without lockstep
+
+With the `lockstep` flag deactivated there is no guarantee when the callback is called. This means the `sim_sensor_update_rate` could not correspond with the real value defined in the world. Inspect the `sim_sensor_update_rate` field, It is likely unable to reach the specified 500 fps. The other low-resolution camera is able to reach the specified value. Take note of the `real time factor` it should be close to 1.0.
```

Comment:
may want to elaborate on what `callback` is, e.g.

```
..there is no guarantee that the camera sensor update happens in the same iteration as the physics simulation.
```

there is also another mention of `callback` below